numba/tests/test_unicode.py

```diff
@@ -338,6 +362,68 @@ def test_find(self, flags=no_pyobj_flags):
                                  cfunc(a, substr),
                                  "'%s'.find('%s')?" % (a, substr))
 
+    def test_count(self):
+        pyfunc = count_usecase
+        cfunc = njit(pyfunc)
+
+        for s in UNICODE_REPEATING_EXAMPLES:
+            extras = ['', ' ', 'xx', s[::-1], s[:-2], s[3:], s, s + s,
+                      'aa', 'ab', 'bb', 'aaa', 'aba']
+            for sub in extras:
```

Comment:
Create `UNICODE_REPEATING_EXAMPLES ` as `STRIP_CASES` and use second field instead `extras` here. Non need to run algorithm on all combinations.